tests/functional/bounced_email.js

```diff
@@ -94,4 +101,46 @@ define([
     }
 
   });
+
+  const setUpBouncedSignIn = thenify(function () {
+    const client = new FxaClient(AUTH_SERVER_ROOT, {
+      xhr: nodeXMLHttpRequest.XMLHttpRequest
+    });
+
+    const email = TestHelpers.createEmail('sync{id}');
+
+    return this.parent
+      .then(clearBrowserState({ force: true }))
+      .then(createUser(email, PASSWORD, { preVerified: true }))
+      .then(openPage(`${config.fxaContentRoot}signin?context=fx_desktop_v2&service=sync`, '#fxa-signin-header'))
+      .then(respondToWebChannelMessage('fxaccounts:can_link_account', { ok: true }))
+      .then(fillOutSignIn(email, PASSWORD))
+      .then(testElementExists('#fxa-confirm-signin-header'))
+      .then(testIsBrowserNotified('fxaccounts:can_link_account'))
+      .then(testIsBrowserNotified('fxaccounts:login'))
+      .then(() => client.accountDestroy(email, PASSWORD))
+      .then(testElementExists('#fxa-signin-bounced-header'));
+  });
+
+  registerSuite({
+    name: 'sign_in with an email that bounces',
+
+    afterEach () {
+      return this.remote.then(clearBrowserState());
+    },
+
+    'click create-account': function () {
+      return this.remote
+        .then(setUpBouncedSignIn())
+        .then(click('#create-account'))
+        .then(testElementExists('#fxa-signup-header'));
```

Comment:
I don't think so, if they're signing up from this state it should definitely be with a different email address.